src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/DatePicker.js

```diff
@@ -182,7 +182,7 @@ class DatePicker extends React.Component<Props> {
         const inputProps = {
             placeholder: placeholder ? placeholder : this.getFormat(),
             valid: valid && !this.showError,
-            disabled: disabled,
+            disabled,
```

Comment:
I personally find this harder to read should we go atleast with [consistent](https://eslint.org/docs/rules/object-shorthand) rule flag /cc @danrot @nnatter

Reply:
not a fan of `consistent` because there are two correct ways for some objects. would prefer `consistent-as-needed` then. but i think the readability is just a matter of getting used to it, so `always` is okay for me too 🙂

Reply:
I am already used to this, think it is easier to read this way, and in many places of the code base it is already handled like this. I trip when I see the same word twice in a line :see_no_evil: